streaming/src/test/scala/org/apache/spark/streaming/util/WriteAheadLogSuite.scala

```diff
@@ -199,6 +198,63 @@ class FileBasedWriteAheadLogSuite
 
   import WriteAheadLogSuite._
 
+  test("FileBasedWriteAheadLog - parallel readAll opens at most 'numThreads' files") {
```

Comment:
This really does not test file reading, just the utility function. Make the name only the name of the method.